jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/SizeLimitHandler.java

```diff
@@ -109,6 +109,7 @@ private class SizeLimitResponseWrapper extends Response.Wrapper
     {
         private final HttpFields.Mutable _httpFields;
         private long _written = 0;
+        private String failed;
```

Comment:
Make this the HttpException itself.  Elsewhere in the code, we create a single failure and return it for all subsequent calls rather than create a new failure each call.

Reply:
done